ui/coordinateTrainer/src/view.ts

```diff
@@ -74,6 +88,40 @@ function side(ctrl: CoordinateTrainerCtrl): VNode {
           )
         ),
       ]),
+      h('form.timeControl.buttons', [
```

Comment:
Mind including a screenshot?

Reply:
I'm not understanding why it's showing "disableTime". Probably because there is no translation.

Reply:
Added to changes ection in PR description.

Reply:
Added to changes section in PR description.